backend/app/models/open_call.rb

```diff
@@ -0,0 +1,22 @@
+class OpenCall < ApplicationRecord
+  extend FriendlyId
+  friendly_id :investor_prefixed_name, use: :slugged
+
+  belongs_to :investor
+
+  validates :instrument_type, inclusion: {in: InstrumentType::TYPES, allow_blank: true}, presence: true
+  validates :ticket_size, inclusion: {in: TicketSize::TYPES, allow_blank: true}, presence: true
+  validates :sdgs, array_inclusion: {in: Sdg::TYPES}
+
+  validates_presence_of :closing_at, :language
+
+  translates :name, :description, :money_distribution, :impact_description
+
+  def investor_prefixed_name
+    "#{investor&.name} #{original_name}"
```

Comment:
And Friendly ID is also kind of new to me :D. Do I understand it correctly that `investor_prefixed_name ` needs to be uniq? If so, It feels a bit tricky to have uniq (slug) for `original_name ` which is localised. Even more, language is just string without any validation so you can put there whatever you want :)

Reply:
Well, I guess I just forgot to add validation for language. I will do.